dnf5-plugins/builddep_plugin/builddep.cpp

```diff
@@ -80,6 +80,7 @@ void BuildDepCommand::set_argument_parser() {
         });
     cmd.register_named_arg(defs);
 
+    auto allow_erasing = std::make_unique<AllowErasingOption>(*this);
```

Comment:
I am really sorry, but  I believe that both option variables (`allow_erasing`, `skip_unavailable`) are defined on incorrect place. I believe  that you should define `allow_erasing` in builddep.hpp `std::unique_ptr<AllowErasingOption> allow_erasing;` (see `downgrade.hpp`).

Reply:
I am sorry, I overlooked that it is already there. Please remove only `auto` from the line.

Reply:
Done. I also moved the variables so it follows the same style of the other files in terms of position.